src/main/resources/standardTestCases/jes_gcs_root.test

```diff
@@ -0,0 +1,14 @@
+name: jes_gcs_root
+testFormat: workflowsuccess
+
+files {
+  wdl: jes_gcs_root/jes_gcs_root.wdl
+  options: jes_gcs_root/jes_gcs_root.options
+}
+
+metadata {
+  workflowName: jes_wdl
+  status: Succeeded
+  "calls.jes_wdl.checkDir.stdout": "gs://cloud-cromwell-dev/centaur-tests/jes_gcs_root/jes_wdl/UUID/call-checkDir/checkDir-stdout.log"
```

Comment:
I would make this more obviously a placeholder, maybe something like `<<UUID>>` or `[[UUID]]`